test/unit/org/apache/cassandra/Util.java

```diff
@@ -850,4 +852,27 @@ public static void setUpgradeFromVersion(String version)
         }
         Gossiper.instance.expireUpgradeFromVersion();
     }
+
+    /**
+     * Sets the length of the file to given size. File will be created if not exist.
+     *
+     * @param file file for which length needs to be set
+     * @param size new szie
+     * @throws IOException on any I/O error.
+     */
+    public static void setFileLength(File file, long size) throws IOException
```

Comment:
nit: unused?

Reply:
thx, picked it up from the refactored 4.1 version. Removing.